packages/maskbook/src/social-network-provider/twitter.com/ui/injectPostInspector.tsx

```diff
@@ -10,7 +10,7 @@ export function injectPostInspectorAtTwitter(current: PostInfo) {
             if (content) {
                 for (const a of content.querySelectorAll('a')) {
                     if (twitterEncoding.payloadDecoder(a.title)) hideDOM(a)
-                    if (a.title === 'http://maskbook.com') hideDOM(a)
+                    if (a.title === 'http://mask.io' || a.title === 'http://maskbook.com') hideDOM(a)
```

Comment:
use regex pattern

Reply:
done

Reply:
@septs No, readability is more important. Regex need more mental cost to understand

Reply:
@Jack-Works  done

Reply:
@Jack-Works ?????

Reply:
I don't find any good reason to prefer regexp. It's harder to read and harder to maintain. If you're not handling a very flexible string match, you don't need it.